src/host/selectionInput.cpp

```diff
@@ -910,19 +910,20 @@ bool Selection::_HandleMarkModeSelectionNav(const INPUT_KEY_INFO* const pInputKe
         }
         else
         {
+            auto d{ _d.write() };

```

Comment:
```suggestion
            auto d{ _d.write() };
            wil::hide_name _d;
```
do we want to hide `_d` like you do in the terminal code?

Reply:
good catch! this is a pretty short block, so I may leave it as is